kakasi

```diff
@@ -74,23 +71,19 @@ def main():
             outfile = a
         elif o in ("-w", "--wakati"):
             wakati = True
-        elif o in ["-H","-K","-J","-E"]:
-            if a in values:
-                mode[o[-1:]] = a
-        elif o in ["-s","-C"]:
-            mode[o[-1:]] = True
-        elif o == "-r":
-            if a in romanvals.keys():
-                mode['r'] = romanvals[a]
-        elif o == "--space":
+        elif o in ("-s","--space"):
             mode["s"] = True
         elif o in ["-S","--separator"]:
             mode["S"] = a
+        elif o in ("-r","-J","-H","-K","-E","-a"):
+            mode[o[1:]] = a
```

Comment:
Here is a problem that we should not use input characters directly into internal value. otherwise all the value should checked.

Example
```
elif o == "-r":
         if a in romanvals.keys():
              mode['r'] = romanvals[a]
```
Your proposal is degraded in this point of view here.